core_backend/tests/api/test_user_tools.py

```diff
@@ -52,6 +59,105 @@ def test_non_admin_create_user(
 
         assert response.status_code == 403
 
+    def test_admin_create_admin_user(
+        self, client: TestClient, fullaccess_token_admin: str
+    ) -> None:
+        response = client.post(
+            "/user/",
+            headers={"Authorization": f"Bearer {fullaccess_token_admin}"},
+            json={
+                "username": "test_username_7",
+                "password": "password",
+                "content_quota": 50,
+                "is_admin": True,
+            },
+        )
+        assert response.status_code == 200
+        json_response = response.json()
+        assert "is_admin" in json_response
+        assert json_response["is_admin"] is True
+
+
+class TestUserUpdate:
+
+    def test_admin_update_user(
+        self, client: TestClient, admin_user: int, fullaccess_token_admin: str
+    ) -> None:
+        content_quota = 1500
+        response = client.put(
+            f"/user/{admin_user}",
+            headers={"Authorization": f"Bearer {fullaccess_token_admin}"},
+            json={
+                "username": TEST_ADMIN_USERNAME,
+                "content_quota": content_quota,
+                "is_admin": True,
+            },
+        )
+
+        assert response.status_code == 200
+
+        response = client.get(
+            "/user/",
+            headers={"Authorization": f"Bearer {fullaccess_token_admin}"},
+        )
+        assert response.status_code == 200
+        json_response = response.json()
+        assert json_response["content_quota"] == content_quota
+
+    def test_admin_update_other_user(
+        self,
+        client: TestClient,
+        user1: int,
+        fullaccess_token_admin: str,
+        fullaccess_token: str,
+    ) -> None:
+        content_quota = 1500
```

Comment:
Here I only updated the content_quota so that the token don't fail after the update if I update the username

Reply:
@lickem22 Is this just so we don't have to take an extra step when performing tests to reset the password manually etc?

Reply:
I am not getting your question. This test is to test the update user function. The update user fuction don't update the password. We would need another endpoint for that

Reply:
Sorry I think I mean token* -> just wasn't sure what would happen if we changed something other than content_qouta.

Reply:
That makes sense now! Thanks for explaining 🙏